lib/src/rive_core/shapes/image.dart

```diff
@@ -36,8 +50,34 @@ class Image extends ImageBase with FileAssetReferencer<ImageAsset> {
     final height = asset!.height;
 
     canvas.save();
-    canvas.transform(worldTransform.mat4);
-    canvas.drawImage(uiImage, ui.Offset(-width / 2, -height / 2), paint);
+    canvas.transform(renderTransform.mat4);
+    if (_mesh == null || !_mesh!.draws) {
+      canvas.drawImage(uiImage, ui.Offset(-width / 2, -height / 2), paint);
+    } else {
+      paint.shader = ui.ImageShader(
+          uiImage,
+          ui.TileMode.clamp,
+          ui.TileMode.clamp,
+          Float64List.fromList(<double>[
```

Comment:
Suggestion : we create some shared helpers for these sorts of matrix44 builders...

Translate(double x, double y)
Scale(double x, double y)
...